src/styles/Editor.css

```diff
@@ -27,16 +27,16 @@
   align-items: center;
   justify-content: flex-start;
   width: 100%;
+  height: 67px;
   border-bottom: 1px white solid;
 }
 
 .editor-expand-panel-arrow {
-  color: #dddcdf;
-  font-weight: bold;
+  color: rgb(39, 33, 52);
   padding: 1rem;
   cursor: pointer;
   flex: 0 1 auto;
-  width: 20px;
+  transition: 1s;
```

Comment:
what does this transition do?

Reply:
Transition was originally added when the bars turned slowly into a cross. But now I have seen the result after removing the transition and since the transition is unnoticeable, I'll remove this.